frontend/src/util/tableHelpers.tsx

```diff
@@ -14,6 +14,17 @@ function comments (comment?: string, languages?: Language[]) {
   ) : null
 }
 
+/**
+ * return full name with format: **LastName**, FirstName
+ * @param firstName 
+ * @param lastName 
+ * @returns 
+ */
+function fullName(firstName: string, lastName: string) {
+  return <p><strong>{lastName}</strong>{`, ${firstName}`}</p>
```

Comment:
do all int always have both? probably

Reply:
interesting, I didn't think of this. So far, all int have both.